suite/tests/client-interface/timer.dll.c

```diff
@@ -49,6 +49,11 @@
  */
 static int buckets[DR_WHERE_LAST];
 
+/* Currently only written to but we may add checks on this once i#4669 failures
```

Comment:
Worth adding an `ASSERT_CURIOSITY` for now?

Reply:
You mean a client version of the core's ASSERT_CURIOSITY?  I think that will just lead to a flaky test if we print something out while we have known failures, right?

Reply:
Oh okay. I don't know whether the `stderr_mask` can filter out those messages, or if we want to do that, or if it's worth it.

Reply:
-stderr_mask filters messages from DR.  For this client we would have to make the expected output file ignore it or something.  Since I was thinking of fixing these failures soon I'm inclined to leave it.